framework/src/media/audio/resample/remix.h

```diff
@@ -0,0 +1,91 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef REMIX_H
+#define REMIX_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+// audio channel masks
+#define CH_MASK_FL                  0x00000001 // Front Left
+#define CH_MASK_FR                  0x00000002 // Front Right
+#define CH_MASK_FC                  0x00000004 // Front Center
+#define CH_MASK_LF                  0x00000008 // Low Frequency
+#define CH_MASK_BL                  0x00000010 // Back Left
+#define CH_MASK_BR                  0x00000020 // Back Right
+#define CH_MASK_FLC                 0x00000040 // Front Left of Center
+#define CH_MASK_FRC                 0x00000080 // Front Right of Center
+#define CH_MASK_BC                  0x00000100 // Back Center
+#define CH_MASK_SL                  0x00000200 // Side Left
+#define CH_MASK_SR                  0x00000400 // Side Right
+#define CH_MASK_TC                  0x00000800 // Top Center
+#define CH_MASK_TFL                 0x00001000 // Top Front Left
+#define CH_MASK_TFC                 0x00002000 // Top Front Center
+#define CH_MASK_TFR                 0x00004000 // Top Front Right
+#define CH_MASK_TBL                 0x00008000 // Top Back Left
+#define CH_MASK_TBC                 0x00010000 // Top Back Center
+#define CH_MASK_TBR                 0x00020000 // Top Back Right
+
+// audio channel layouts
```

Comment:
Below can be changed to enum & moved to mediatype.h to open user.
And then we can make api for layout masking.(But I coudn't find usage of mask in this PR)

Reply:
layout is a combinations of channel mask, so I put them together here.

Reply:
If move to MediaTypes.h, how about channel mask?

Reply:
It's also moved to cpp, will open to user if necessary in future.